server/controllers/overviewController.ts

```diff
@@ -90,21 +84,18 @@ export default class OverviewController {
       offencesOverview,
       nonAssociationSummary,
     ] = await Promise.all([
-      this.userService.getStaffRoles(clientToken, res.locals.user),
       pathfinderApiClient.getNominal(prisonerNumber),
       manageSocCasesApiClient.getNominal(prisonerNumber),
       this.offencesService.getNextCourtHearingSummary(clientToken, bookingId),
       this.offencesService.getActiveCourtCasesCount(clientToken, bookingId),
       showCourtCaseSummary ? this.offencesService.getLatestReleaseCalculation(clientToken, prisonerNumber) : null,
-      prisonerInCaseLoad ? this.moneyService.getAccountBalances(clientToken, bookingId) : null,
-      prisonerInCaseLoad || isPomOrReceptionUser
+      permissions.money?.view ? this.moneyService.getAccountBalances(clientToken, bookingId) : null,
+      permissions.adjudications?.view
         ? this.adjudicationsService.getAdjudicationsOverview(clientToken, bookingId)
         : null,
-      prisonerInCaseLoad ? this.visitsService.getVisitsOverview(clientToken, bookingId, prisonerNumber) : null,
+      permissions.visits?.view ? this.visitsService.getVisitsOverview(clientToken, bookingId, prisonerNumber) : null,
```

Comment:
This has changed from just requiring `prisonerInCaseLoad` to `isPrisonerInCaseLoad || userHasRoles([Role.PomUser, Role.ReceptionUser], userRoles)` 

https://github.com/ministryofjustice/hmpps-prisoner-profile/pull/658/files#diff-436cd1a03284961f6db546c6d570d7f2d99d017aacdcf407b84fb925c79cfa47R60

Is this right?

Reply:
No, looks like an oversight